### PR TITLE
[CAP-212] Creation of App using React-Native.

### PR DESCRIPTION
**Pull request by lpelizzaMS**
### Background

This is a first approach to the technology React Native. The app is called “Albums”. This app gets albums information from an endpoint Rest and shows the information on the app like a List.

###Features

- Show list of albums
- Redirect to Amazon to buy the album selected.

###Screenshot
rn-albums

### CODE REVIEW

File: ReactNative/Albums/src/components/AlbumList.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    console.log(this.state);
+
+    return (
+      <ScrollView>
+        {this.renderAlbums()}
+      </ScrollView>
+    );
+  }
+}
+
```
Comment (mravinale) [inline ReactNative/Albums/src/components/AlbumList.js:30]: good start!! 👍

Comment (mravinale) [reply ReactNative/Albums/src/components/AlbumList.js:30]: I think we can ignore all compiled files, and mantian only react-native code and configurations what do you think?

**Review by mmaldini33 — COMMENTED** [review]
🥇 🥇  Nice job! Congratulations! 🥇 🥇

**Review by mravinale — APPROVED** [review]

**Comment by mravinale** [comment]
let's go with redux! 
next PR :)

**Pull request merged**